python/tvm/relay/op/nn/nn.py

```diff
@@ -481,7 +481,8 @@ def global_avg_pool2d(data,
 def upsampling(data,
                scale=1,
                layout="NCHW",
-               method="NEAREST_NEIGHBOR"):
+               method="NEAREST_NEIGHBOR",
+               align_corners=True):
```

Comment:
Since nearest is the default for upsampling, I think it makes more sence if the default for align_corners is False.

Reply:
I agree and have changed the defaults for upsampling.